users/gurtatiLND/website/webpage.html

```diff
@@ -74,15 +73,16 @@ <h3>My Skills</h3>
                 <div class="main-container">
                     <h2 class="heading projects">PROJECT</h2>
                     <!--at this part I will talk about game which I will build for this application -->
-                    <p class="intro">Here you will find a collection of my personal projects, each accompanied by its own case study. I created these projects for my portfolio and to practice and build my skills.</p>
+                    <p class="intro">Here you will find a project I created both for the application and to practice and build my skills.</p>
                     <div class="projects-content">
                         <div class="projects-row">
                             <div class="projects-row-img-content">
-                                <img src="./assets/first_memorygame.png" class="projects-row-img" alt="wepapp screenshot">
+                                <img src="./assets/RPS_game.png" class="projects-row-img" alt="wepapp screenshot">
                             </div>
                             <div class="projects-row-content">
                                 <h3>Game</h3>
-                                <p>A simple flip-card memory game using React. The game is designed to train memory by having players flip cards to find matching pairs. It features a clean UI, state management for game logic, and uses React components to handle the card flipping and matching mechanics.</p>
+                                <p>The Rock, Paper, Scissors game is a simple yet entertaining game where both participants(user and computer) choose either rock, paper, or scissors. The game can end in a tie or result in one player winning and the other losing. I plan to build this game with HTML, CSS, and JavaScript.</p>
+                                <!-- CHANGE THE LINK TO THE GAME -->
                                 <a class="btn" href="https://shimmering-muffin-69285d.netlify.app/">View Project</a>
```

Comment:
This is still linking to the memory game rather than the rock paper scissors game

Reply:
hi yeah it's not ready yet so I will change the link when I finish the game so hopefully in the next PR.